db/db.go

```diff
@@ -81,12 +81,18 @@ func OpenWithDSN(dbPath, dsn string) (*DB, error) {
 
 // OpenInMemory opens an in-memory database.
 func OpenInMemory() (*DB, error) {
-	return open(fqdsn(":memory:", ""))
+	return OpenInMemoryWithDSN("mode=memory&cache=shared")
```

Comment:
See the comments in the commit message: https://github.com/rqlite/rqlite/pull/279/commits/9c670e6b9e747adf3ff992d86374ff6b6f678db7